engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -205,7 +205,9 @@ def status_code(error_message)
 
         def has_no_regcode?(auth_header)
           auth_header ||= '='
+          # rubocop:disable Style/SlicingWithRange
           auth_header = auth_header[(auth_header.index('=') + 1)..-1]
```

Comment:
if I understood correctly, so this checks for an `=` sign in `auth_header` and retrieve anything after it.

Probably `auth_header.split('=', 2).last.empty?` is a better choice?